### DISCUSSION TITLE
Welcome to Nungeon Discussions!

### DISCUSSION THREAD

**Discussion by Goniix** [Goniix/Nungeon#23] [General]
zub

**Comment by Goniix** [comment]
![player](https://user-images.githubusercontent.com/87310572/191925382-e9c9279a-2509-44a5-aa7e-e16304b4622c.png)
![Background](https://user-images.githubusercontent.com/87310572/191925406-687733e2-e507-472f-a486-b631b643d249.png)

**Reply by pioupia** [reply]
Mec t'es trop chaud !

**Comment by pioupia** [comment]
merci
